cli.mjs

```diff
@@ -18,16 +19,86 @@ let OUTPUT_ROOT; // The root in which data is saved
 const INVALID_ACTION_ERROR = new Error("Invalid action: " + ACTION);
 const INVALID_DEPLOYMENT_ERROR = new Error("Invalid deployment: " + DEPLOYMENT);
 
+/** -------------------- COMMANDER -------------------- */
+const commander = new Command();
+// default: [download | delete ] not provided, run main() as usual continuing with prompting
+commander.action(() => {});
+
+// download: optional argument studyID and participantID skips relative prompts
+commander
+  .command(`download`)
+  .argument(`[studyID]`)
+  .argument(`[participantID]`)
+  .description(`Download experiment data from Firebase provided study ID and participant ID`)
+  .action((studyID, participantID) => {
+    ACTION = "download";
+    STUDY_ID = studyID;
+    PARTICIPANT_ID = participantID;
+  });
+
+// delete: optional argument studyID and participantID skips relative prompts
+commander
+  .command(`delete`)
+  .argument(`[studyID]`)
+  .argument(`[participantID]`)
+  .description(`Delete experiment data from Firebase provided study ID and participant ID`)
+  .action((studyID, participantID) => {
+    ACTION = "delete";
+    STUDY_ID = studyID;
+    PARTICIPANT_ID = participantID;
+  });
+
+// register: optional argument studyID and participantID skips relative prompts
+commander
+  .command(`register`)
+  .argument(`[studyID]`)
+  .argument(`[participantID]`)
+  .description(`Register new partipant under study provided a partipantID and studyID`)
+  .action((studyID, participantID) => {
+    ACTION = "register";
+    STUDY_ID = studyID;
+    PARTICIPANT_ID = participantID;
+  });
+
+commander.parse();
+
+// print message if download or delete provided, along with optional args provided
+if (ACTION != undefined) {
+  console.log(
+    `${ACTION} data from Firebase ${STUDY_ID == undefined ? "" : `given study ID: ${STUDY_ID}`} ${PARTICIPANT_ID == undefined ? "" : `and participant ID: ${PARTICIPANT_ID}`}`
+  );
+}
```

Comment:
Can you actually move this to the very top of the `main` function? I think that will make it a little more obvious that this is the very first code that is actually executed when the script is run